api-update/README.md

```diff
@@ -0,0 +1,26 @@
+## API Update lambdas
+There will be a separate project for each different file metadata update message as well as a common project for shared code. So for now, this is one for the antivirus, the file format and the checksum. 
+
+### Running the tests 
+This is all from the `api-update` directory
+
+To run the common tests
+
+`sbt test`
+
+To run the other project tests, e.g. for the antivirus
+
+`sbt antivirus/test`
+
+To build the jar
+
+`sbt antivirus/assembly`
+
+To run the tests in IntelliJ, you will need to set environment variables. These are set in the sbt file which isn't read by intellij
+```
+"API_URL" -> "http://localhost:9001/graphql" 
+"AUTH_URL" -> "http://localhost:9002/auth" 
```

Comment:
Because this is a lambda function which basically just runs a single method with arguments defined by the lambda, there's no way I can think of to use a config file, short of trying to run an akka server inside a lambda but that's probably overkill.
These are static as well, they're defined in the test config.

Reply:
Does the code get deployed to the Lambda as a JAR? If you're using [typesafe config](https://github.com/lightbend/config), I'm 90% sure you can drop an application.conf file into src/main/resources (or test/main/resources) and load it with `ConfigFactory.load()`.

Just thought it would make life easier for anyone running the tests for the first time.

Reply:
If we use environment variables, we can set the url and keycloak secret ones in terraform based off the terraform resources so if anything changes, it will update automatically. The URLs are unlikely to change but the keycloak secret might. The keycloak secret would have to populated from environment variables anyway as you can't commit that to a repo.
You can also encrypt environment variables

Reply:
You can still use environment variables. You'd just be reading them in config file rather than in the code. It would just make it easier to override them in the tests without having to set them in IntelliJ.

Reply:
application.conf looks good. 👍 Does that mean the readme can be updated?

Reply:
Yep, I've removed the section on setting the environment variables.